src/Proto.Actor/IMailbox.cs

```diff
@@ -77,27 +78,31 @@ public class DefaultMailbox : IMailbox
     {
         private readonly IMailboxQueue _systemMessages;
         private readonly IMailboxQueue _userMailbox;
+        private readonly IMailboxStatistics[] _stats;
         private IDispatcher _dispatcher;
         private IMessageInvoker _invoker;
 
         private int _status = MailboxStatus.Idle;
         private bool _suspended;
 
-        public DefaultMailbox(IMailboxQueue systemMessages, IMailboxQueue userMailbox)
+        public DefaultMailbox(IMailboxQueue systemMessages, IMailboxQueue userMailbox, params IMailboxStatistics[] stats)
         {
             _systemMessages = systemMessages;
             _userMailbox = userMailbox;
+            _stats = stats;
```

Comment:
Either we should have something like this:
`_stats= stats ?? Array.Empty<IMailboxStatistics>()`
Or, we should default empty array to null.

Im not sure how foreach behaves over empty arr, if that still results in some compiler magic and a null check might be faster (?)
Optimize for default path.

Thoughts?